src/StaticAssets/src/ResponseHeader.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+
+namespace Microsoft.AspNetCore.StaticAssets;
+
+// Represents a response header for a static resource.
+[DebuggerDisplay($"{{{nameof(GetDebuggerDisplay)}(),nq}}")]
+internal class ResponseHeader(string name, string value)
+{
+    public string Name { get; } = name;
+    public string Value { get; } = value;
+
+    private string GetDebuggerDisplay() => $"Name: {Name} Value:{Value}";
```

Comment:
Nit (I know this only affects us), but:
```suggestion
    private string GetDebuggerDisplay() => $"Name: {Name} Value: {Value}";
```

Reply:
Same nit for `StaticAssetDescriptor`.

Reply:
This also shows up in the customers debugger sessions, so it's better if it displays correctly.